### PR TITLE
Fbfeedfragment

### LINKED ISSUE #116: Add Facebook feed on the home page
**Is your feature request related to a problem? Please describe.**
Add facebook feed on the home activity of the application. It should fetch all the posts made by iit patna pages and display them.

### PR DESCRIPTION
**Pull request by ravi-kishan**
Fixes #116 

Changes: 
Added fragment to show facebook feed
Added support classes for fragment injection

Screenshots for the change:

### CODE REVIEW

File: app/src/main/java/com/iitp/njack/iitp_connect/core/facebook/feed/FacebookFeedAdapter.java [hunk 1]
```diff
@@ -16,19 +15,17 @@
...
-        this.glide = glide;
     }
 
     @NonNull
     @Override
     public FacebookFeedViewHolder onCreateViewHolder(@NonNull ViewGroup viewGroup, int position) {
         FacebookFeedViewHolder facebookFeedViewHolder =
             new FacebookFeedViewHolder(DataBindingUtil.inflate(LayoutInflater.from(viewGroup.getContext()),
-                R.layout.facebook_post_item, viewGroup, false), viewGroup.getContext(), glide);
+                R.layout.facebook_post_item, viewGroup, false),viewGroup.getContext());
```
Comment (dr0pdb) [inline app/src/main/java/com/iitp/njack/iitp_connect/core/facebook/feed/FacebookFeedAdapter.java]: space after comma please.

File: app/src/main/java/com/iitp/njack/iitp_connect/core/facebook/feed/FacebookFeedFragment.java [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+import javax.inject.Inject;
+
+import static com.iitp.njack.iitp_connect.ui.ViewUtils.showView;
+
+
+public class FacebookFeedFragment extends Fragment implements FacebookFeedView, Injectable {
+
+    @Inject
+    ViewModelProvider.Factory viewModelFactory;
```
Comment (dr0pdb) [inline app/src/main/java/com/iitp/njack/iitp_connect/core/facebook/feed/FacebookFeedFragment.java]: explicitly define it as public.

**Review by dr0pdb — CHANGES_REQUESTED** [review]
Good work!
Just a few minor changes.

File: app/src/main/java/com/iitp/njack/iitp_connect/core/facebook/feed/FacebookFeedFragment.java [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+import com.iitp.njack.iitp_connect.databinding.FragmentFacebookFeedBinding;
+import com.iitp.njack.iitp_connect.ui.ViewUtils;
+
+import java.util.List;
+
+import javax.inject.Inject;
+
+import static com.iitp.njack.iitp_connect.ui.ViewUtils.showView;
+
+
```
Comment (sheetal-g9) [inline app/src/main/java/com/iitp/njack/iitp_connect/core/facebook/feed/FacebookFeedFragment.java:33]: Remove unnecessary extra line.

Comment (dr0pdb) [reply app/src/main/java/com/iitp/njack/iitp_connect/core/facebook/feed/FacebookFeedFragment.java:33]: That's okay. There are too many formatting issues in the project. I'll sort them out all at once. I'll merge this one.

**Review by dr0pdb — APPROVED** [review]

**Pull request merged**